proposed_pr.py

```diff
@@ -0,0 +1,107 @@
+from datetime import timedelta, datetime
+import pandas as pd
+
+from instore.models import Sensor
+
+from .metric import Metric
+from .metric_definitions import SENSOR_TYPE_METRICS
+
+
+class CliShot(object):
```

Comment:
Bad name. Shot was initial name on instore module, and this client replaces Shot/Metric classes, so no need to name it as CliSomething all classes of the module.
Four us can make sense, but not for next developers